config/vufind/searches.ini

```diff
@@ -153,6 +153,13 @@ retain_filters_by_default = true
 ; commented out, fallback ID lookups will be disabled.
 ;fallback_id_field = previous_id_str_mv
 
+; This setting can be used to configure pagination control on top of results.
+; Possible options are:
+; empty string or false  No top pagination (default)
+; simple                 Simple next/prev button pagination
+; full                   Full pagination alike to the one at the bottom of results
+;top_paginator = simple
```

Comment:
I've now tested with Blender, EDS and Primo, and can't see any reason why it wouldn't work with others as well, so I went ahead and added it, and tried to follow the existing convention in each files regarding comments.

If you really want to, you can at least hide the bottom navigation with css. But I wouldn't want to encourage that, since I can't see that being a good idea in normal cases.